soccer/main.py

```diff
@@ -14,16 +15,45 @@
 LEAGUE_IDS = leagueids.LEAGUE_IDS
 TEAM_NAMES = teamnames.team_names
 
+
+def write_config_file():
+    """Write API key to config.py file"""
+    print("No API key found!")
+    print("Please visit {0} and get an API token.".format(BASE_URL))
+    while True:
+        confkey = raw_input("Please enter API key: ")
+        if len(confkey) == 32:  # 32 chars
```

Comment:
assuming this solution is accepted, are you sure the API will always be 32 characters?